kroxylicious-kubernetes-api/src/main/resources/META-INF/fabric8/kafkaproxies.kroxylicious.io-v1.yml

```
@@ -61,6 +64,12 @@ spec:
                             type: string
                           type: object
                       type: object
+                replicas:
+                  description: "The number of instances of this proxy to run"
```

Maybe we should make the pod-per-instance model clearer here?

```suggestion
                  description: "The number of proxy Pods to run"
```.